draft-ietf-quic-transport.md

```diff
@@ -2916,6 +2938,34 @@ PATH_CHALLENGE frame previously sent by the endpoint, the endpoint MAY generate
 a connection error of type UNSOLICITED_PATH_RESPONSE.
 
 
+## NEW_TOKEN frame {#frame-new-token}
+
+An server sends a NEW_TOKEN frame (type=0x10) to provide the client a token to
+send in a the header of an Initial packet for a future 0-RTT connection.
+
+The NEW_TOKEN frame is as follows:
```

Comment:
Just an idea: Maybe it would be more consistent if we use the NEW_TOKEN frame in the Retry packet. No strong feelings in either direction.

Reply:
@marten-seemann Keeping the Retry packet simple, i.e., without frames, seems best, as then DoS devices don't need any frame logic. They can just generate tokens and parse Initial token headers.

Reply:
It would basically just be one byte more (the type byte). But I think I'm convinced we shouldn't make it a frame. It eliminates one failure mode (i.e. wrong type byte).